cmd/secret-wrapper/main.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+
+	coreclientset "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	"github.com/openshift/ci-tools/pkg/util"
+)
+
+const (
+	DIR = "/tmp/secret"
+)
+
+func main() {
+	if err := run(os.Args); err != nil {
```

Comment:
Why not use `entrypoint`?

Reply:
There isn't much overlap other than the very simple function that executes a program, and we would still need a separate container to create the secret when `entrypoint` finishes.